src/OrbitGl/App.cpp

```diff
@@ -345,32 +345,36 @@ void OrbitApp::OnTracepointEvent(orbit_client_protos::TracepointEventInfo tracep
       is_same_pid_as_target);
 }
 
-void OrbitApp::UpdateModulesAbortIfModuleWithoutBuildIdReloader(
+void OrbitApp::UpdateModulesAbortCaptureIfModuleWithoutBuildIdNeedsReload(
     absl::Span<const ModuleInfo> module_infos) {
-  auto updated_modules = module_manager_->AddOrUpdateModules(module_infos);
+  auto updated_modules = module_manager_->AddOrUpdateNotLoadedModules(module_infos);
```

Comment:
More expressive name here instead of updated_modules. Maybe `modules_unable_to_update` or `changed_modules_without_build_id`.

Reply:
done